### DISCUSSION TITLE
Alternatives to MLX90614ESF-BCC-000-TU for STELLA-2.0

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#16] [Build Notes: How do I make a STELLA?]
The thermal infrared surface temperature sensor (MLX90614ESF-BCC-000-TU) at 35º FOV for STELLA-2.0 is not available in stock. Is MLX90641ESF-BCB-000-SP a good alternative? It uses voltage supply 3V ~ 3.6V instead of 2.6V ~ 3.6V. If not, can anyone suggest any alternative component?

**Comment by Landsat-Mike** [comment]
There are some in stock at this retailer here: https://www.futureelectronics.com/p/semiconductors--analog--sensors--temperature/mlx90614esf-bcc-000-tu-melexis-2532877

**Reply by chadoulis** [reply]
Yeap, I am aware of this site. However this site requires a minimum order of 47 pieces. I was wondering if there is any cost-effective alternative.

**Comment by paulmirel** [comment]
Hi chadoulis, 

The TU part number means it comes in a tube. The SP part number means it comes in a sample pack. Digikey has them, for sale but the single unit. https://www.digikey.com/en/products/detail/melexis-technologies-nv/MLX90614ESF-BCC-000-SP/5416203

**Comment by paulmirel** [comment]
Mouser has 754 of them: https://www.mouser.com/ProductDetail/Melexis/MLX90614ESF-BCC-000-SP Though they show the wrong photo, the part description has the correct part number and field of view.
